kcron-core/src/commonMain/kotlin/com/ucasoft/kcron/core/builders/Builder.kt

```diff
@@ -208,19 +206,20 @@ class Builder(firstDayOfWeek: WeekDays = WeekDays.Monday) {
         return null
     }
 
+    @Suppress("UNCHECKED_CAST")
```

Comment:
Cast could be checked in this function.

Reply:
For me it's a problem relative to the management od generics with your definitions of `fun minusDays(days: Int): CronDateTime<T>` inside `CronDateTime<out T>`.
It sounds too restrictive to have a good typing  for the return type of plusDays/minusDays .